pandas/core/groupby/groupby.py

```diff
@@ -1982,7 +2055,7 @@ def _get_cythonized_result(self, how, grouper, aggregate=False,
                 result = algorithms.take_nd(obj.values, result)
 
             if post_processing:
-                result = post_processing(result)
+                result = post_processing(result, inferences)
```

Comment:
FYI this will fail if we try to pass in `post_processing` with `pre_processing` as `inferences` will be undefined in that case. Since it's all internal I suppose not a big deal for now, but could also add an explicit check at the start of the function if it doesn't add too much bloat.